contributing/documentation/translations.rst

```diff
@@ -1,87 +1,6 @@
 Translations
 ============
 
-The Symfony documentation is written in English and many people are involved
-in the translation process.
+Featuring translations of the Symfony documentation `has been terminated`_.
```

Comment:
This seems a bit harsh, and I know some people are sensitive about this decision. Something like this:

>
The Symfony documentation is not officially translated, though some community groups still
maintain some translations. For more information, see <link>.